cpu/kinetis_common/wdog.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2015 PHYTEC Messtechnik GmbH
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_kinetis_common_wdog
+ *
+ * @{
+ *
+ * @file
+ * @brief       Low-level WDOG driver implementation
+ *
+ * @author      Johann Fischer <j.fischer@phytec.de>
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include "periph_conf.h"
+
+#ifndef KINETIS_WDOG_ADVANCED
+/**
+ * Attempts to determine the type of the WDOG,
+ * using the WDOG_STCTRLH_CLKSRC_MASK field.
+ */
+#ifdef WDOG_STCTRLH_CLKSRC_MASK
```

Comment:
Missing `#include "cpu.h"` or `#include "cpu-conf.h"` in order to include the CPU specific header so that this test makes sense.